src/main.rs

```diff
@@ -60,6 +60,30 @@ fn print_err_and_exit(err: js_errors::JSError) {
   std::process::exit(1);
 }
 
+fn display_file_info(filename: String, dir: &deno_dir::DenoDir) {
+  let maybe_out = dir.code_fetch(&filename, ".");
+  if maybe_out.is_err() {
+    println!("{}", maybe_out.unwrap_err());
+    return;
+  }
+  let out = maybe_out.unwrap();
+
+  println!("local filename: {}", &(out.filename));
```

Comment:
Let's try to make these labels (text before colon) into single words without qualifiers

`filename:`